### PR TITLE
netd should log version info

### PR DESCRIPTION
**Pull request by cadmuxe**

### CODE REVIEW

File: scripts/install-cni.sh [hunk 1]
```diff
@@ -15,6 +15,9 @@
 # limitations under the License.
 
 set -u -e
+
+echo "netd version: @VERSION@"
```
Comment (varunmar) [inline scripts/install-cni.sh]: This is fine, but FYI the other substitution variables below use a single @ sign before the name.

File: build/build.sh [hunk 2]
```diff
@@ -31,6 +31,8 @@ if [ -z "${VERSION}" ]; then
     exit 1
 fi
 
+cat scripts/install-cni.sh | sed "s/@VERSION@/${VERSION}/" > scripts/install-cni-build.sh
```
Comment (varunmar) [inline build/build.sh]: Suggest removing the unnecessary use of cat:
sed -e "s/@VERSION@/${VERSION}/g" scripts/install-cni.sh > scripts/install-cni-build.sh
Also consider if we want to surround the substitution string with @ or just at the beginning. I'm okay with either one, but let's be consistent.

Comment (cadmuxe) [reply build/build.sh]: Oh, it got merged before I update it, will do it in the next pr.
Thanks.

**Pull request merged**